kratos/processes/simple_mortar_mapper_wrapper_process.h

```diff
@@ -91,11 +91,25 @@ class SimpleMortarMapperProcessWrapper
     {
         // The default parameters
         Parameters default_parameters = GetDefaultParameters();
+        const bool origin_are_conditions_is_defined = ThisParameters.Has("origin_are_conditions");
+        const bool destination_are_conditions_is_defined = ThisParameters.Has("destination_are_conditions");
         ThisParameters.ValidateAndAssignDefaults(default_parameters);
 
         // Automatic detect the entities
-        if (rOriginModelPart.NumberOfElements() > 0) ThisParameters["origin_are_conditions"].SetBool(false);
-        if (rDestinationModelPart.NumberOfElements() > 0) ThisParameters["destination_are_conditions"].SetBool(false);
+        if (!origin_are_conditions_is_defined) {
+            if (rOriginModelPart.NumberOfElements() > 0) {
+                ThisParameters["origin_are_conditions"].SetBool(false);
+            } else if (rOriginModelPart.NumberOfConditions() == 0) {
+                KRATOS_ERROR << "No conditions defined on origin model part" << std::endl;
+            }
+        }
```

Comment:
What happens if my MP has both conditions and elements (e.g. shell)?

Reply:
You should define it by hand, this is the automatic part

Reply:
Define what by hand?

Reply:
Well, if you want to keep the this check I'm fine (we assume that the user provides two model parts ready for mapping).  However, I'd throw a warning to inform about the change in the  `origin_are_conditions` from `True` to `False`.